SETUP.md

```diff
@@ -1 +1,37 @@
+# Getting setup
 
+We will be using Anaconda to install and manage a Python 'environment' for us to work in. Rather than using any pre-installed Python 
+interpretors this has the advantage that it will not conflict with any existing libraries, and everyone will be using the same verisons.
+
+## 1. Install Anaconda
+
+If you haven't already got Anaconda installed, download and install it from: https://www.continuum.io/downloads. Choose Python 3.5 as this 
+is the latest version of Python.
+
+## 2. Create a new environment for the workshop
+
+Now you have installed Anaconda we can create an evironment to work in. Open your terminal (or 'Anaconda Prompt' on Windows) and type:
+
+``` $ conda create -c conda-forge -n python_workshop cis jupyter```
```

Comment:
The other choice is to use it as a worked example, and not suggest they run it themselves immediately (or not to worry if they can't). The idea of that part is to demonstrate some practical OO so I don't want to spend too long on it.